### ISSUE TITLE
Add URL field to playground interface

### ISSUE THREAD

**Issue by karenell** [mapbox/search-playground#20]
### Context

The old search api playground made it easy to see the full request URL which made troubleshooting query parameters straightforward and reflects what users would do IRL. 

![screen shot 2018-10-22 at 3 40 25 pm](https://user-images.githubusercontent.com/34380259/47314890-f602ce80-d610-11e8-820b-857fe6a9a3ee.png)

The new API playground doesn't include a field like this, and the URL is only ever rendered in a long string that's hard to parse, like:

```
https://www.mapbox.com/search-playground/#{"url":"","index":"mapbox.places","staging":false,"onCountry":true,"onType":true,"onProximity":true,"onBBOX":true,"onLimit":true,"onLanguage":true,"countries":[],"proximity":"","typeToggle":{"country":false,"region":false,"district":false,"postcode":false,"locality":false,"place":false,"neighborhood":false,"address":false,"poi":false},"types":[],"bbox":"","limit":"","autocomplete":true,"languages":[],"languageStrict":false,"onDebug":false,"selectedLayer":"","debugClick":{},"query":"san fran"}
```

Here's an example of how tilequery playground gets everything on one page as well:

https://www.mapbox.com/help/tilequery-api-playground/

### Request
Add a place where the full request URL is rendered.

**Comment by kbauhaus** [comment]
Thanks for ticketing, @karenell! After playing around in the search-playground interface a bit, I found that if you click on the second icon from the left at the top of the page, it will open up the API request in a new tab. A user could grab the full request URL from that new tab, though I'd advocate for displaying the API request URL in the the search-playground itself, perhaps above the other settings. 

<img width="580" alt="screen shot 2018-10-22 at 12 56 38 pm" src="https://user-images.githubusercontent.com/11279680/47316074-1a53b080-d5fb-11e8-81c3-4a8c0cd1bb60.png">


This was an advantage of the previous search playground - generating and displaying the API request URL helps users familiarize themselves with how filtering with query parameters works:
<img width="579" alt="screen shot 2018-10-22 at 1 02 51 pm" src="https://user-images.githubusercontent.com/11279680/47315973-d2348e00-d5fa-11e8-9d5e-a9e2d24da63e.png">


Separately, this also reminded me that as a user it would be useful to have text appear on hover for each of the icons at the top right of the UI. It'd be ideal to know before clicking that the far-left icon will display the full API response, etc.

<img width="580" alt="screen shot 2018-10-22 at 12 56 38 pm" src="https://user-images.githubusercontent.com/11279680/47316128-3e16f680-d5fb-11e8-9ea0-afb2d5b6e3d0.png">

**Comment by ingalls** [comment]
@kbauhaus  ![screenshot from 2018-10-25 14-16-16](https://user-images.githubusercontent.com/1297009/47521286-9f8bcf00-d860-11e8-83b3-a8fed71c449e.png)

This should appear in hey.mapbox.com before the end of day. Waiting for the new publisher version and then will hopefully deploy publically by EOW

**Comment by ingalls** [comment]
![screenshot from 2018-10-25 14-34-52](https://user-images.githubusercontent.com/1297009/47522313-3d809900-d863-11e8-9ac8-5f95c99cdc84.png)

![screenshot from 2018-10-25 14-34-17](https://user-images.githubusercontent.com/1297009/47522317-407b8980-d863-11e8-92f0-7f94f2c480f2.png)

@karenell Going to close this as complete. Feel free to reopen a new ticket if you run into and problems.

This should be visible on hey.mapbox.com by the end of the day and the public facing mapbox.com by the end of the week.
